src/main/resources/templates/db_scripts/updateItemStatusDate.sql

```diff
@@ -0,0 +1,23 @@
+
+-- Updates item status date if status name was changed.
+-- Item status date stores with timezone.
+CREATE OR REPLACE FUNCTION ${myuniversity}_${mymodule}.update_status_date() RETURNS TRIGGER
+AS $$
+  DECLARE
+	  newStatus text;
+  BEGIN
+  	newStatus = NEW.jsonb->'status'->>'name';
+  IF newStatus IS NULL THEN  RETURN NEW;
```

Comment:
Will it work for the case when OLD.status != null but NEW.status is null?

Reply:
I just added "IS DISTINCT FROM" and removed this check.